libraries/core/src/main/java/com/chesire/nekome/core/settings/ApplicationSettings.kt

```diff
@@ -49,6 +51,23 @@ class ApplicationSettings @Inject constructor(
             return status
         }
 
+    /**
+     * Gets the default [HomeScreenOption] the home screen that shows after login
+     */
+    val defaultHomeScreen: HomeScreenOptions
```

Comment:
Need to add a unit test for this property

Reply:
Added the unit test cases for `defaultHomeScreen`